packages/playwright-core/src/client/eventEmitter.ts

```diff
@@ -258,8 +301,6 @@ export class EventEmitter implements EventEmitterType {
       for (let i = listeners.length - 1; i >= 0; i--)
```

Comment:
don't  we need to copy the listeners array here?

Reply:
We are removing them all synchronously and you are not supposed to interact with event emitter during the service 'removeListener' events...